app/build.gradle

```diff
@@ -108,7 +108,7 @@ android {
             buildConfigField "boolean", "multiuser", "true"
             buildConfigField "boolean", "activeStockFragment", "true"
             buildConfigField "boolean", "patientTestedByDefault", "true"
-            buildConfigField "String", "AuthoritiesProvider", "\"" + applicationId + ".data.database.utils.SCMMDBProvider\""
```

Comment:
applicationId is necessary to avoid provider conflict with buddybuild app installation, Why you need remove it?